x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.test.tsx

```diff
@@ -120,34 +120,6 @@ describe('EnginesList', () => {
 
 describe('CreateEngineButton', () => {
   describe('disabled={true}', () => {
-    it('renders a disabled button that shows a popover when focused', () => {
```

Comment:
I repurposed the popover for the platinum license acknowledgement based on the design.
The initial platinum license popover was only visible when not on an platinum+/trial license (and hidden on trial/platinum+)
The `Create Search Application` would also be disabled when not on platinum+/trial.
Because the popover is now visible regardless of the license level and whether the create button is disabled, I removed part of the tests.